assets/components/headers/simpleHeader/simpleHeader.scss

```diff
@@ -3,50 +3,138 @@
 
 .component-simple-header {
   background-color: gu-colour(header-dark);
+  display: flex;
+  align-items: center;
+  justify-content: stretch;
+  overflow: hidden;
 
   .svg-guardian-logo {
-    height: 100%;
+    display: block;
+    float: right;
+    margin-top: $gu-v-spacing / 2.4;
+    margin-bottom: $gu-v-spacing * 1.5;
+    height: 48px;
+    @include mq($from: mobileLandscape) {
+      height: 56px;
+    }  
+    @include mq($from: tablet) {
+      height: 72px;
+    }  
+    @include mq($from: desktop) {
+      height: 95px;
+    }  
   }
 }
 
-.component-simple-header__content {
-  max-width: gu-span(12);
-  padding: ($gu-v-spacing/4) ($gu-h-spacing / 2) ($gu-v-spacing /2);
-  height: 42px;
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-
-  @include mq($from: mobileMedium) {
-    height: 52px;
-  }
-
-  @include mq($from: mobileLandscape) {
-    height: 68px;
-  }
+.component-simple-header__wrapper {
+  width: 100%;
+  margin: auto;
+  padding: 0 ($gu-h-spacing / 2);
+  position: relative;
+  flex: 1 1 auto;
 
   @include mq($from: tablet) {
-    height: 86px;
-    padding: ($gu-v-spacing/2) ($gu-h-spacing / 2) $gu-v-spacing;
+    max-width: gu-span(11);
   }
 
   @include mq($from: leftCol) {
-    max-width: gu-span(13);
+    max-width: gu-span(14) - ($gu-h-spacing);
   }
 
   @include mq($from: wide) {
-    max-width: gu-span(14);
+    max-width: gu-span(16) - ($gu-h-spacing);
   }
 }
 
-.component-simple-header__utility {
-  float: left;
-  height: 100%;
-  flex: 0 0 auto;
+.component-simple-header-topnav {
+  display: flex;
+  justify-content: space-between
 }
 
-.component-simple-header__link {
+.component-simple-header-topnav__utility {
+  padding-top: $gu-v-spacing / 2;
+}
+
+.component-simple-header-topnav-logo {
+  z-index: 10;
+  position: relative;
+  @include mq($from: tablet) {
+    width: gu-span(5);
+    background-color: gu-colour(header-dark);
+    float: right;
+    .component-simple-header--one-row & {
+      border-left: 1px solid gu-colour(header-border);
+    }
+  }
+  @include mq($from: wide) {
+    padding-right: gu-span(1) + $gu-h-spacing / 2;
+  }
+}
+
+.component-simple-header-topnav-logo__graun {
   float: right;
   height: 100%;
   flex: 0 1 auto;
 }
+
+.component-simple-header-bottomnav {
+  display: none;
```

Comment:
why do we have the `display: none;`? isn't it overriden by the very next line?

Reply:
yes 😹